packages/api/src/entities/AppConfig.ts

```diff
@@ -0,0 +1,172 @@
+/* istanbul ignore file */
+import { Entity, Property, EntityManager } from '@mikro-orm/core';
+import { ConstructorValues } from '../utils/types';
+import { Node } from './Node';
+
+export enum ConfigKey {}
+
+type JSONValue = string | number | boolean | Record<string, any> | any[];
+
+export type AppConfigConstructorValues = ConstructorValues<AppConfig>;
+type AppConfigPropertyKeys = keyof AppConfigConstructorValues;
+
+@Entity()
+export class AppConfig extends Node<AppConfig> {
```

Comment:
@johnkahn if we don't extend `Node` here and simply create a brand new entity, will `persistAndFlush` act as an `upsert` when we try and save a new `AppConfig` with the same `key`?